lib/vssh/libssh2.c

```diff
@@ -3167,10 +3167,13 @@ static CURLcode ssh_connect(struct Curl_easy *data, bool *done)
 
   sshc = &conn->proto.sshc;
 
-#ifdef CURL_LIBSSH2_DEBUG
   if(conn->user) {
-    infof(data, "User: %s", conn->user);
+    infof(data, "User: '%s'", conn->user);
+  }
+  else {
+    infof(data, "User: NULL");
```

Comment:
Would it make sense to do the same for `Password:`, for consistency?

Reply:
Good question, this slots in with the two final "further thought" points. I'll leave this to the maintainers to make the call.

Though the reason I did not do it is I considered the latter a purely debug print (i.e. compile-time flag).

Reply:
It'd be nice to keep this consistent IMO.